corehq/apps/domain/tests/test_delete_domain.py

```diff
@@ -1102,3 +1102,17 @@ def test_hard_delete_cases_none_to_delete(self):
 def ensure_deleted(domain):
     if domain._rev:
         domain.delete()
+
+
```

Comment:
should this add the unit testing only decorator?

Reply:
I don't think so. It's in a test module. Production code should never import from test modules.